lib/packagekit-glib2/pk-task.c

```
@@ -1227,6 +1250,61 @@ pk_task_remove_packages_async (PkTask *task, gchar **package_ids, gboolean allow
+ * @progress_user_data: data to pass to @progress_callback
+ * @callback_ready: the function to run on completion
+ * @user_data: the data to pass to @callback_ready
+ *
+ * Remove a package (optionally with dependancies), along with any configuration files, from the system.
+ * If @allow_deps is set to %FALSE, and other packages would have to be removed,
+ * then the transaction would fail.
+ *
+ * Since: 0.5.2
+ **/
+void
+pk_task_purge_packages_async (PkTask *task, gchar **package_ids, gboolean allow_deps, gboolean autoremove, GCancellable *cancellable,
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'pk_task_purge_packages_async' of similar type ('gboolean *') are easily swapped by mistake

(bugprone-easily-swappable-parameters)